code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -120,6 +121,527 @@ datum/reagent/toxin/slimejelly/on_mob_life(var/mob/living/M as mob)
+	else
+		holder.present_machines[1] += (500- holder.present_machines[1])
+	holder.remove_reagent(src.id,5)
+datum/reagent/toxin/emote
+	name = "Pure Emotium"
+	id = "emote"
+	description = "This shouldn't be difficult to figure out."
+	color = "#75AC53"
+datum/reagent/toxin/emote/on_mob_life(var/mob/living/M as mob)
+	if(prob(20))
+		M.emote(pick("fart","flap","aflap","airguitar","blink","shrug","cough","sneeze","shake","twitch"))
+		M.hallucination += 1
+datum/reagent/toxin/smokep
+	name = "Smoke Powder"
+	id = "smokep"
+	description = "Try farting I dare you!"
+	color = "#75AC53"
+datum/reagent/toxin/smokep/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
+		M.emote(pick("cough","sneeze"))
+datum/reagent/toxin/smokep/on_fart(var/mob/living/M as mob)
+	M.visible_message("<span class='danger'>[M] farts out a noxious cloud!</span>")
+	create_smoke(src, 10)
+
+datum/reagent/toxin/fart//for wu eric
+	name = "Fartium"
+	id = "fart"
+	description = "Hmm I wonder?"
+	color = "#10AC53"
+datum/reagent/toxin/fart/on_mob_life(var/mob/living/M as mob)
+	if(prob(20))
+		M.emote("fart")//yup thats it
+datum/reagent/toxin/sboom
+	name = "Nitrogenated isopropyl alcohol"
+	id = "sboom"
+	description = "Hmm , needs more nitrogen!"
+	color = "#13BC5E"
+datum/reagent/toxin/superboom
+	name = "N-amino azidotetrazole"
+	id = "superboom"
+	description = "If this stuff is even slightly warmed it will make a mess."
+	color = "#13BC5E"
+datum/reagent/toxin/superboom/on_mob_life(var/mob/living/M as mob)
+	if(M.bodytemperature > 285)//you CAN avoid it , just it's difficult
+		var/location = get_turf(holder.my_atom)
+		explosion(location,volume/20,volume/15,volume/10)
+		holder.clear_reagents()
+		return
+
+
+datum/reagent/toxin/superboom/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
+	if(M.bodytemperature > 285)
+		var/location = get_turf(holder.my_atom)
+		explosion(location,volume/20,volume/15,volume/10)
+		holder.clear_reagents()
+		return
+datum/reagent/toxin/superboom/on_update()
+	if(!holder)
+		return
+	if(holder.present_machines[1] >= 285)
+		var/location = get_turf(holder.my_atom)
+		explosion(location,volume/20,volume/15,volume/10)
+		holder.clear_reagents()
+		return
+
+datum/reagent/toxin/ethyl
+	name = "Ethyl alcohol"
+	id = "ethyl"
+	description = "Brilliant at burning"
+	color = "#A245B2"
+datum/reagent/toxin/carbonf
+	name = "Carbonic fluoride"
+	id = "carbonf"
+	description = "A fairly nasty chemical used to produce potent medicines"
+	color = "#A300B3"
+datum/reagent/toxin/carbonf/on_mob_life(var/mob/living/M as mob)
+	M.adjustToxLoss(1)
+datum/reagent/toxin/sparky
+	name = "Electrostatic substance"
+	id = "sparky"
+	description = "The lights glow nearby when this substance is around"
+	color = "#A300B3"
+
+datum/reagent/toxin/spark/on_mob_life(var/mob/living/M as mob)
+	if(prob(10))
+		M.adjustFireLoss(3)//extremely weak damage
+		var/datum/effect/effect/system/spark_spread/sparks = new /datum/effect/effect/system/spark_spread
```

Comment:
shoot people with this when the station is being plasma flooded for maximum !!FUN!!